bitnami/redis/Chart.yaml

```
@@ -24,4 +24,4 @@ maintainers:
 name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
-version: 16.7.0
+version: 16.7.1
```

As we are implementing a new feature to the chart, we should bump its version to a new minor.

```suggestion
version: 16.8.0
```

---

Updated